app/views/organizations/staff/pets/show.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render DashboardPageComponent.new(crumb: :dashboard_pet, crumb_options: [@pet]) do |c| %>
   <% c.with_header_title do %>
```

Comment:
Can you please align the new badge on the center?
<img width="174" alt="image" src="https://github.com/user-attachments/assets/3463e849-27c2-4ab6-ae6b-ac55fd5c9966">